frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -116,7 +116,16 @@ const plugin: Plugin<ConsumedExtensions> = [
       detect: detectManagedODF,
     },
     flags: {
-      required: [OCS_FLAG],
+      required: [OCS_MODEL_FLAG],
+    },
+  },
+  {
+    type: 'FeatureFlag/Custom',
+    properties: {
+      detect: detectManagedODF,
+    },
+    flags: {
+      required: [ODF_MODEL_FLAG],
```

Comment:
We need to add extension for both cases, right ?? 
When only ODF is installed and when only OCS is installed.

Reply:
Let's do for both